osu.Framework/Platform/GameHost.cs

```diff
@@ -243,61 +243,72 @@ public void Collect()
         private void unhandledExceptionHandler(object sender, UnhandledExceptionEventArgs args)
         {
             var exception = (Exception)args.ExceptionObject;
-            exception.Data["unhandled"] = "unhandled";
-            handleException(sender, exception);
+
+            logException(exception, "unobserved");
```

Comment:
think you meant `unhandled` here?
```suggestion
            logException(exception, "unhandled");
```